Python/bytecodes.c

```diff
@@ -4199,6 +4207,24 @@ dummy_func(
             frame->instr_ptr = (_Py_CODEUNIT *)instr_ptr;
         }
 
+        tier2 op(_DEOPT, (--)) {
+            EXIT_TO_TIER1();
+        }
+
+        tier2 op(_SIDE_EXIT, (--)) {
+            EXIT_TO_TRACE();
+        }
+
+        tier2 op(_ERROR_0, (--)) {
+            GOTO_UNWIND();
+        }
+
+        tier2 op(_ERROR_N, (values[oparg] --)) {
+            (void)values;
+            SYNC_SP();
+            GOTO_UNWIND();
```

Comment:
Hm. Maybe this should just be a `_STACK_SHRINK` op and we just follow it with an `_ERROR` instruction. That way we only have one error instruction instead of two.

Reply:
We can just drop `_ERROR_0` it is the same as `_ERROR_N` with `oparg==0`.